docs/transaction-groups_spec.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html><html lang="en"><head><title>transaction-groups</title><link rel="stylesheet" href="spec_table.css"></head><body><table><caption>Specification for transaction-groups</caption><tr>
```

Comment:
we only want the html changes for the schemas that change

Reply:
:+1: